js/parts/Tooltip.js

```diff
@@ -8,9 +8,25 @@
  *
  * */
 'use strict';
+var __read = (this && this.__read) || function (o, n) {
```

Comment:
See if we can avoid the code that makes TypeScript include this polyfill. Likely candidate is array destructuring.